presentational/Translation.js

```diff
@@ -38,17 +72,18 @@ const TranslationResult = ({
       <>
         <TitleBar>{word}</TitleBar>
         <Translations>
-          {translationForms.map(({ translations }) =>
-            translations.map(({ from, toType, to, example }, index) => (
+          {translationForms
+            .map(({ translations }) => translations)
+            .flat()
+            .map(({ from, toType, to, example }, index) => (
               <ResultRow
                 from={from}
                 toType={toType}
                 to={to}
                 example={example}
-                key={index}
+                index={index}
```

Comment:
Does it work to skip key?

Reply:
good catch, it doesn't—I didn't check the console